scripts/generate_cmake.py

```diff
@@ -1,129 +0,0 @@
-#!/usr/bin/env python
```

Comment:
We should also remove this line: https://github.com/uxlfoundation/oneMath/blob/develop/.github/CODEOWNERS#L14

Reply:
Good catch. Updated.